src/content/developers/docs/consensus-mechanisms/index.md

```diff
@@ -48,51 +46,63 @@ A consensus mechanism in a cryptoeconomic system also helps prevent certain kind
 
-### Sybil resistance & chain selection {#sybil-chain}
+### Sybil resistance {#sybil-chain}
+
+Have you ever seen a website that's being spammed by a ton of fake accounts? These fake accounts are run by bots which try to sell you something. To get around that, a lot of sites employ a system called "captchas". These captchas are difficult for computers to solve but easy for humans, preventing automation from making fake identities. Another way to prevent fake accounts is to force users to pay for a service. This payment associates a cost with automation, preventing armies of bots from taking over a site.
+
+Blockchains have a form of captcha of their own. This process of preventing fake identities from taking over a network is called "Sybil resistance". Like captcha or premium accounts on a website, Sybil resistance mechanisms associate some kind of work or value to participating in the blockchain. This causes an innate investment in the success of the network.
+
+**Sybil resistance** describes how a protocol fares against a [Sybil attack](https://en.wikipedia.org/wiki/Sybil_attack). Sybil attacks are when one user or group pretends to be many users. Resistance to this type of attack is essential for a decentralized blockchain and enables miners and validators to be rewarded equally based on resources put in. Proof-of-work and proof-of-stake protect against this by making users expend a lot of energy or put up a lot of collateral. These protections are an economic deterrent to Sybil attacks.
+
+Proof-of-work and proof-of-stake are not consensus protocols by themselves, but since each blockchain needs some form of Sybil resistance, it's easy to conflate the two concepts. They actually are Sybil resistance mechanisms and block author selectors. They are a way to decide who is the author of the latest block. It's this Sybil resistance mechanism combined with proposal mechanism that makes up a truly decentralized consensus protocol.
+
+### Proposal mechanism {#proposal-mechanism}
 
-Now technically, proof-of-work and proof-of-stake are not consensus protocols by themselves, but they are often referred to as such for simplicity. They are actually Sybil resistance mechanisms and block author selectors; they are a way to decide who is the author of the latest block. It's this Sybil resistance mechanism combined with a chain selection rule that makes up a true consensus mechanism.
+A proposal mechanism is what's used to determine the next block proposed on the network. In Nakamoto consensus, a person who mines a winning nonce gets the right to be the leader for the next block. They propose that block to the network. In Eth2, a random beacon decides the order of proposers from the list of available validators. In both cases, after a block is proposed, it's up to the next proposer to decide if they want to build on that block or on another block proposed in the case of competing proposals. This is what's known as the "chain selection rule".
```

Comment:
Again, we should eliminate usage of "Eth2" here. Perhaps `Under a proof-of-stake protocol,` instead of `In Eth2,`